### PR TITLE
new file:   packages/odnnr/odnnr.2.0.0/opam

### PR DESCRIPTION
**Pull request by UnixJunkie**

### CODE REVIEW

File: packages/odnnr/odnnr.2.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+"""
+post-messages:
+  """
+Please interact with R to install required dependencies in user-space:
+R
+install.packages('keras', repos='http://cran.r-project.org')
+library(keras)
+install_keras(method = 'conda', tensorflow = 'cpu')""" {failure}
+extra-files: [
+  ["install_keras.r" "md5=b07326eec56b36df7a2aba08b29264e5"]
```
Comment (kit-ty-kate) [inline packages/odnnr/odnnr.2.0.0/opam:40]: This file seems missing from the PR

**Comment by UnixJunkie** [comment]
Sorry, I added the missing file

**Comment by avsm** [comment]
Could you test this works locally with `opam install .` please?  Fails because its trying to run `R CMD files/<...>` but the extra-files will drop it in the current directory (without the `files/`)

**Comment by UnixJunkie** [comment]
I removed the files/ prefix.

File: packages/odnnr/odnnr.2.0.0/opam [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+homepage: "https://github.com/UnixJunkie/odnnr"
+bug-reports: "https://github.com/UnixJunkie/odnnr/issues"
+dev-repo: "git+https://github.com/UnixJunkie/odnnr.git"
+license: "LGPL-2.1-only with OCaml-LGPL-linking-exception"
+install: ["R" "CMD" "BATCH" "install_keras.r"]
+build: ["dune" "build" "-p" name "-j" jobs]
+depends: [
+  "ocaml"
+  "conf-r"
+  "dune" {>= "1.0.0"}
```
Comment (mseri) [inline packages/odnnr/odnnr.2.0.0/opam:13]: The dune-project file requires dune 1.11. I can modify the file unless you want to lower the bound in the sources.
```suggestion
  "dune" {>= "1.11.0"}
```

**Comment by mseri** [comment]
The CI is acting weird, and Travis is missing the R packages. I think the best way forward is to merge, since the metadata now seem correct, and then fix it if some issue comes out by using the package

**Pull request merged**